lib/pipeline.js

```diff
@@ -135,6 +135,82 @@ Pipeline.prototype = {
   },
 
   /**
+   * Bind to a streams events, and re-emit them on the pipeline
+   * @param {Stream} stream
+   * @param {String|Array} events
+   * @returns {Pipeline}
+   * @chainable
+   */
+  bind( stream, events ) {
```

Comment:
Hmmm, so it sounds like _effectively_ these are actually methods on streams themselves, with the pipeline only being a parameter used in the `bind` call? Would it make sense to make them static methods on the Pipeline class (instead of being instance methods) to make it clearer that none of the pipeline's internal state is being modified? \*shrug\*  Or would that mess up the API too much / make it too awkward?

> Nah, that'd prevent unbinding of events after a stream's been removed from the pipeline and such.

Hmmm? Doesn't the usage of `unbindAll` in `splice` mean that a removed stream automatically has all its events unbound?